csm/conf/configure.py

```diff
@@ -53,11 +53,11 @@ async def execute(self, command):
             Log.error(f"Configuration Loading Failed {e}")
         self._set_deployment_mode()
         try:
+            uds_public_ip = command.options.get('uds_public_ip')
```

Comment:
Mostly this section should not be required now. it should be taken from configuration file. Lets discuss it in detail to understand both side (provisioning restriction and UDS configuration) better.

Reply:
This PR has been updated, please review again.